src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.ImportExport.cs

```diff
@@ -0,0 +1,558 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Most of this seemed to be static methods which would apply equally well to ECDH; can it be factored off to a helper class?